src/App.js

```diff
@@ -52,22 +52,25 @@ function allowTabs() {
 }
 
 export default function App() {
-    const classes = useStyles();
+  const BACKEND_URL = `https://cjsbackdev.herokuapp.com/`
+  const classes = useStyles();
 
     // set state
     const [timer, setTimer] = useState(null);
     const [input, setInput] = useState("");
     const [output, setOutput] = useState("");
     const [inputLanguage, setInputLanguage] = useState("js");
     const [outputLanguage, setOutputLanguage] = useState("py");
+    const [supportedLanguages, setSupportedLanguages] = useState([]);
 
-    async function wakeUpServer() {
-        fetch(`https://cjsback.herokuapp.com/`)
+    async function getRequest() {
```

Comment:
I would prefer having this function return the languages instead of setting state itself. Would make it easier to test and remove a side-effect.